src/classes/HH_NameSpec_TEST.cls

```diff
@@ -396,6 +402,51 @@ private with sharing class HH_NameSpec_TEST {
     /*********************************************************************************************************
     @description
     operation:
+        Test Household Name, and Formal/Informal greetings when Contact API names are case insensitive
+    verify:
+        HH Name, Formal Greeting, and Informal Greeting are correct
+    **********************************************************************************************************/            
+    static testMethod void testHHNamingWhenContactApiNamesAreCaseInsensitive() {
```

Comment:
Thanks David. I believe it is very important for the test name to indicate what is being tested and expected. The reasons are mentioned above (readability is the main one), but most importantly tests should be self-documenting, and developer should not guess what is being tested when a test is run from Developer Console or any other app. I will decrease the test names, since they are too long.